_includes/markdown/React.md

```diff
@@ -0,0 +1,130 @@
+The purpose of this document is to help guide you through working with a potential project using the JavaScript library, [React](https://reactjs.org/). Outside of The official React documentation, you should find here personalized recommendations for using the library based on the types of projects we typically  see at 10up. We have broken these recommendations out into common elements we tend to interact with often in client work. If something you’re looking for isn’t represented you can either submit a pull request to this repository or refer to the [official React documentation](https://reactjs.org/).
+
+## Figuring Out If React Is Right For Your Project
+
+When deciding whether React is the right tool for your project, it might help to ask the following questions:
+
+* Will this project involve building a user interface that requires statefulness? In other words, will your user interface need to...react...in real time to user actions or changeable data coming from an API?
+* Does your project require greater flexibility than a full-fledged MVC or MV(star) framework would allow? For example: does your application need to exist within a larger site context, rather than forming the basis for the entire site?
```

Comment:
I think this needs more direction on where React is the best choice or not based on the answer to the question.

Reply:
I'm also unsure about this question. There have been times where I felt like React was a great choice even though I was only building one section of a website, maybe something like a plugin dashboard in the admin.

Reply:
The biggest thing in using React right now is the weight of the library. So bringing it in for a small component on a site probably isn't the right way to go if we can accomplish the same result with something smaller like Vue. I agree that this question is a little confusing though. Maybe we can just lose it since it doesn't seem to help much.